src/cbl-mariner/2.0/crossdeps/amd64/Dockerfile

```diff
@@ -2,6 +2,9 @@ FROM mcr.microsoft.com/cbl-mariner/base/core:2.0
 
 RUN tdnf update -y && \
     tdnf install -y \
+        # Needed to install MicroBuild in Azure DevOps pipelines
+        sudo \
```

Comment:
As a rule, we don't want to install `sudo` in new images. I would hope that the MicroBuild script would check to see whether the `nuget` package is installed before installing it. In that case, the right thing to do would be to install the `nuget` package here instead.